apps/files/src/components/TemplatePreview.vue

```diff
@@ -124,6 +124,9 @@ export default {
 		onFailure() {
 			this.failedPreview = true
 		},
+		focus() {
+			this.$el.querySelector('input').focus()
```

Comment:
You can also add a `ref="input"` to the input element in the template and then simply

```suggestion
			this.$refs.input?.focus()
```